src/Symfony/Component/DependencyInjection/Tests/Fixtures/containers/container14.php

```diff
@@ -4,8 +4,14 @@
 
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 
-class ProjectServiceContainer extends ContainerBuilder
-{
+/**
+ * Be careful here: this class is included in Tests\Dumper\GraphvizDumperTest::testDumpWithFrozenCustomClassContainer
```

Comment:
see also #13947 :) (though I think it's easier to merge this and close the other PR)